src/openrct2/Editor.h

```diff
@@ -85,6 +85,12 @@ extern "C"
 
     void editor_open_windows_for_current_step();
 
+    /**
+    * Removes all unused objects from the object selection.
+    * @return The number of removed objects.
+    */
+    sint32 editor_remove_unused_objects();
```

Comment:
This doc-block should be on the actual definition, not on this declaration.

Reply:
No, this is right. It has been done else where, see a lot of the core header files and it also says so in the code style wiki page.

Reply:
Oh, ok. Sorry about that, @oli414 .